client/v2/client.go

```diff
@@ -405,29 +407,106 @@ func NewPointFrom(pt models.Point) *Point {
 	return &Point{pt: pt}
 }
 
+// makeChunks splits blob string into chunks that size
+// of each chunk does not exceed given chunkSize
+// makeChunks assumes that size of blob string is greater than chunkSize
+// makeChunks splits blob by selecting chunk by range
+func makeChunks(blob string, chunkSize int) []string {
+	var (
+		size        = int(math.Ceil(float64(len(blob)) / float64(chunkSize)))
+		chunks      = make([]string, 0, size)
+		start, stop int
+	)
+
+	// range by chunkSize
+	for i := 0; i < len(blob); i += chunkSize {
+		// set boundaries of range
+		stop = start + chunkSize
+
+		// if the stop is more than blob length
+		// then set it to end of the blob
+		if stop > len(blob) {
+			stop = len(blob)
+		}
+
+		// move dividing position back until do not match a comma character
+		for l := stop; l > 0 && l != len(blob); l-- {
+			stop = l
+			if string(blob[l]) == "," {
+				break
+			}
+		}
+
+		// if the current position is a comma
+		// then move starting position forward to skip comma sign
+		if string(blob[start]) == "," {
+			start += 1
+		}
+
+		chunks = append(chunks, blob[start:stop])
+		start = stop
```

Comment:
This won't work if a single field exceeds `chunkSize` in length, because `stop` will find its way right back to `start`, then repeat for however many chunks are expected to be found after this point.